docs/reference/commandline/volume_ls.md

```diff
@@ -168,10 +168,10 @@ Valid placeholders for the Go template are listed below:
 
 Placeholder   | Description
 --------------|------------------------------------------------------------------------------------------
-`.Name`       | Network name
-`.Driver`     | Network driver
-`.Scope`      | Network scope (local, global)
-`.Mountpoint` | Whether the network is internal or not.
+`.Name`       | Volume name
+`.Driver`     | Volume driver
+`.Scope`      | Volume scope (local, global)
+`.Mountpoint` | Whether the volume is internal or not.
```

Comment:
This description is a bit odd; Perhaps "The mount point of the volume on the host" ?